comment/messages.py

```diff
@@ -86,3 +88,13 @@ class FlagState:
 class FollowError:
     EMAIL_REQUIRED = _('Email is required to subscribe {model_object}')
     SYSTEM_NOT_ENABLED = _('Subscribe system must be enabled')
+
+
+class BlockState:
+    UNBLOCKED = _('Unblocked')
+    BLOCKED = _('Blocked')
+
+
+class BlockedUserMSG:
```

Comment:
To make this consistent with the other class names in this module
```suggestion
class BlockUserMessage:
```